source/includes/_callbacks.md

```diff
@@ -47,6 +48,27 @@ var callstats = new callstats();
 callstats.initialize(AppID, AppSecret, localUserId, csInitCallback, csStatsCallback);
 ```
 
-The `initialize()` API authenticates the javascript WebRTC application with the callstats.io back-end, and sets up a trusted relationship with it. The API is extended by adding a `csStatsCallback` parameter. The callback parameter is OPTIONAL.
+The csStatsCallback function can either be given as a parameter to the initialize() call, or set separately with the on() functionality. The `initialize()` API authenticates the javascript WebRTC application with the callstats.io back-end, and sets up a trusted relationship with it. The API is extended by adding a `csStatsCallback` parameter. The callback parameter is OPTIONAL.
 
 The `csStatsCallback()` will be called by the callstats.js for each PeerConnection independently at regular intervals. By default the interval is set as 10 seconds to make sure we do not overwhelm the app with too many messages. For more information, please check out our blog on [`csStatsCallback()`] (http://www.callstats.io/2015/08/24/statscallback-webrtc-media-quality-status/)
+
+## The default configuration callback
+
+```javascript
+function csDefaultConfigurationCallback(config) {
+  var pc = new RTCPeerConnection(config.peerConnection);
+  getUserMedia(config.media);
+
+}
+```
+
+The csDefaultConfigurationCallback function is set with the on() functionality. The callback is invoked when the default configuration provided in the callstats.io dashboard is available. It is only fired once.
```

Comment:
The config has the following format: {peerConnection: (javascript object), media: (javascript object)} (dont know how you tell these).
Also, you could link these parameters to the webrtc specification, so people know which format they should take
- peerConnection object is of type RTCConfiguration https://www.w3.org/TR/webrtc/#rtcconfiguration-dictionary
- media object is of type MediaStreamConstraints https://www.w3.org/TR/mediacapture-streams/#mediastreamconstraints